perun-contract/src/Perun/Error.hs

```diff
@@ -66,10 +72,65 @@ data FindChannelException
     -- TODO: This most likely has to be removed, since we cannot prevent
     -- anyone from just sending UTXOs to our channel.
```

Comment:
```suggestion
```

IMO this is resolved, as we only throw this in `findChannel`, which trys to find a channel without verifying the channel token and channel parameters / id.
We sort of deprecated this, as the actual contract endpoints only use `findChannel'`, aka. finding a channel with verification of channel-token. Consequently, they will not throw this anymore.

Reply:
Okay, I am fine with that assessment.